example/rbac/create_role.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ETCD_OPERATOR_ROOT=$(dirname "${BASH_SOURCE}")/../..
+
+ROLE_NAME="${ROLE_NAME:-etcd-operator}"
+ROLE_BINDING_NAME="${ROLE_BINDING_NAME:-etcd-operator}"
+NAMESPACE="${NAMESPACE:-default}"
+CREATE_CRD="${CREATE_CRD:-true}"
```

Comment:
Just assume `CREATE_CRD=true` here and only handle that.

Reply:
Great, that simplifies things.